shared/oae/api/oae.api.util.js

```diff
@@ -589,8 +589,11 @@ define(['exports', 'require', 'jquery', 'underscore', 'oae.api.config', 'markdow
             if (options.submitHandler && $.isFunction(options.submitHandler)) {
                 submitCallback = options.submitHandler;
             }
+            
+            // Register common custom validation methods
+            $.validator.addMethod('displayname', isValidDisplayName, require('oae.api.i18n').translate('__MSG__NAMES_CANNOT_INCLUDE_URLS_OR_AT_SIGNS__'));
```

Comment:
Rather than making this validator globally available, I'd prefer to add it as a custom validator inside of the `userdetails` widget. The `register` widget has some good examples on how to do this as part of `setUpValidation`